ui/src/pages/version/components/forms/components/transformer/components/table_outputs/UpiPreprocessOutputCard.js

```diff
@@ -67,7 +67,13 @@ export const UpiPreprocessOutputCard = ({ output, onDelete, onChangeHandler, err
           <ColumnsComboBox
             columns={output.transformerInputTableNames || []}
             onChange={onChange("0.upiPreprocessOutput.transformerInputTableNames")}
-            title="Transformer input table names"
+            title={
+              <EuiToolTip content="Transformer input table names contains tables that can be used to enrich prediction_table using standard transformer">
```

Comment:
I think it should be
`List of table names that will be used to populate prediction_table field in the request to model`